src/components/SQLEditor/SQLEditor.tsx

```diff
@@ -98,77 +195,52 @@ function SQLEditor({ execCallback, value, results }: Params) {
       historyToUse = history.slice(0, tempHistory.length + 2);
     }
     const ret = historyToUse.pop();
-    localStorage.setItem(SQL_HISTORY_TEMP_CONTENT_KEY, JSON.stringify(historyToUse));
+    localStorage.setItem(
+      SQL_HISTORY_TEMP_CONTENT_KEY!,
+      JSON.stringify(historyToUse),
+    );
     return ret;
   };
 
-  const formatEditorContents = () => {
-    if (editorContents && ace) {
-      const formatted = formatSQL(editorContents, { language: 'postgresql' });
+  const formatSql = () => {
+    if (sql && ace) {
+      const formatted = formatSQL(sql, { language: 'postgresql' });
       ace.getSession().setValue(formatted);
       ace.focus();
     }
   };
 
-  const annotate = (): Ace.Annotation[] | undefined => {
-    if (!ace) {
-      return;
-    }
-    if (!results) {
-      return;
-    }
+  // onChange method
+  const onValueChange = (newValue: string) => {
+    // It updates current state and call onChange
+    setSql(newValue);
 
-    if (Array.isArray(results)) {
-      const err = results.find(r => r.error);
-      if (!err) {
-        return;
-      }
-      const q = err.original_query;
-      if (!q) {
-        return;
-      }
-      // we take the 1st 100 lines and see if any of them are contained in the
-      // query that failed. A rather naive approach.
-      const lines = ace.getSession().getLines(0, 100);
-      const trimmed = q.trim();
-      const pos = lines
-        .map((v, i) => {
-          const search = v.trim();
-          if (search.length > 0 && trimmed.startsWith(search)) {
-            return i;
-          }
-          return -1;
-        })
-        .find(v => v > 0);
-      if (pos) {
-        return [
-          {
-            row: pos,
-            column: 0,
-            text: err.error?.message || '',
-            type: 'error',
-          },
-        ];
-      }
-      return;
-    }
-
-    if (results.error) {
-      // We always show the 1st line and 1st column as the source of the error.
-      return [
-        {
-          row: 0,
-          column: 0,
-          text: results.error.message,
-          type: 'error',
-        },
-      ];
+    if (onChange) {
+      onChange(newValue);
     }
   };
 
   return (
-    <div className="max-w-screen-xl">
+    <div className="w-full">
```

Comment:
I think this made the width out-of-sync with the results table:
![image](https://github.com/crate/crate-gc-admin/assets/4594061/76316ae1-f166-4275-bd97-579e514a91b7)

Should we set the same thing on the results table too so that it's full screen?